lua/telescope/previewers/init.lua

```diff
@@ -199,26 +207,18 @@ previewers.qflist = term_previewer.qflist
 ---         -- for example `search` and `matchadd`
 ---       end)
 ---     to achieve that.
----   - If you want to read a file into the buffer its best to use
+---   - If you want to read a file into the buffer it's best to use
 ---     `buffer_previewer_maker`. But access this function with
 ---     `require('telescope.config').values.buffer_previewer_maker`
 ---     because it can be redefined by users.
----@see previewers.new_buffer_previewer
```

Comment:
I actually thought i added `@see` to the grammar but apparently this breaks things. Hmm.
Seems like i need to look into this

Reply:
Opened issue upstream

Reply:
https://github.com/tjdevries/tree-sitter-lua/issues/7